ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/model/jsonrpc/PrivateTransaction.java

```diff
@@ -0,0 +1,260 @@
+  }
+
+  public Optional<ValueHolder<String>> getGas() {
+    return gas;
+  }
+
+  public Optional<ValueHolder<String>> getTo() {
+    return to;
+  }
+
+  public Optional<ValueHolder<String>> getValue() {
+    return value;
+  }
+
+  public Optional<ValueHolder<String>> getData() {
+    return data;
+  }
+
+  public Optional<ValueHolder<String>> getPrivateFrom() {
+    return privateFrom;
+  }
+
+  public Optional<ValueHolder<List<String>>> getPrivateFor() {
+    return privateFor;
+  }
+
+  public Optional<ValueHolder<String>> getRestriction() {
+    return restriction;
+  }
+
+  public Optional<ValueHolder<String>> getPrivacyGroupId() {
+    return privacyGroupId;
+  }
+
+  public static Builder defaultTransaction() {
+    return new PrivateTransaction.Builder()
+        .withFrom(UNLOCKED_ACCOUNT)
+        .withNonce("0xe04d296d2460cfb8472af2c5fd05b5a214109c25688d3704aed5484f9a7792f2")
+        .withGasPrice("0x9184e72a000")
+        .withGas("0x76c0")
+        .withTo("0xd46e8dd67c5d32be8058bb8eb970870f07244567")
+        .withValue(DEFAULT_VALUE)
+        .withData(
+            "0xd46e8dd67c5d32be8d46e8dd67c5d32be8058bb8eb970870f072445675058bb8eb970870f072445675")
+        .withPrivateFrom(PRIVATE_FROM)
+        .withPrivateFor(singletonList(PRIVATE_FOR))
+        .withRestriction("restricted");
+  }
+
+  public static class Builder {
+    private Optional<ValueHolder<String>> from = Optional.empty();
+    private Optional<ValueHolder<String>> nonce = Optional.empty();
+    private Optional<ValueHolder<String>> gasPrice = Optional.empty();
+    private Optional<ValueHolder<String>> gas = Optional.empty();
+    private Optional<ValueHolder<String>> to = Optional.empty();
+    private Optional<ValueHolder<String>> value = Optional.empty();
+    private Optional<ValueHolder<String>> data = Optional.empty();
+    private Optional<ValueHolder<String>> privateFrom = Optional.empty();
+    private Optional<ValueHolder<List<String>>> privateFor = Optional.empty();
+    private Optional<ValueHolder<String>> restriction = Optional.empty();
+    private Optional<ValueHolder<String>> privacyGroupId = Optional.empty();
+
+    public Builder withFrom(final String from) {
+      this.from = createValue(from);
+      return this;
+    }
+
+    public Builder withNonce(final String nonce) {
+      this.nonce = createValue(nonce);
+      return this;
+    }
+
+    public Builder missingNonce() {
+      this.nonce = Optional.empty();
+      return this;
+    }
+
+    public Builder withGasPrice(final String gasPrice) {
+      this.gasPrice = createValue(gasPrice);
+      return this;
+    }
+
+    public Builder missingGasPrice() {
+      this.gasPrice = Optional.empty();
+      return this;
+    }
+
+    public Builder withGas(final String gas) {
+      this.gas = createValue(gas);
+      return this;
+    }
+
+    public Builder missingGas() {
+      this.gas = Optional.empty();
+      return this;
+    }
+
+    public Builder withTo(final String to) {
```

Comment:
I know this is generated by automatic builder generator, but this one should be renamed to `withReceiver(final String to)` :). similarky `withMissingReceiver`